config/routes.rb

```diff
@@ -6,6 +6,7 @@
   post '/students/import' => 'students#create_multiple'
   resources :students
   patch '/students/:student_id/teachers/:id' => 'students_teachers#update', as: :student_teacher
+  patch '/students/:student_id/teachers/:id/remove' => 'students_teachers#remove', as: :remove_student_teacher
```

Comment:
This should be a delete route because it's deleting an association. Doesn't need a new "as:" because it's the same as the update:

`delete '/students/:student_id/teachers/:id' => 'students_teachers#destroy'`